_topic_maps/_topic_map.yml

```
@@ -21,6 +21,8 @@ Distros: openshift-service-mesh
 Topics:
 - Name: Installing OpenShift Service Mesh
   File: ossm-installing-openshift-service-mesh
+- Name: Red Hat OpenShift Service Mesh and cert-manager
+  File: ossm-cert-manager-assembly
```

must match the [context](https://github.com/openshift/openshift-docs/pull/83665/files#diff-f2252ef1f126521d2cbec4deaf02fc1e683c3d8201594021f90fb6bc91e516f3R5)
```suggestion
  File: ossm-cert-manager
```